include/version.h.in

```diff
@@ -2,5 +2,5 @@
 #define GRASS_VERSION_MAJOR    @GRASS_VERSION_MAJOR@
 #define GRASS_VERSION_MINOR    @GRASS_VERSION_MINOR@
 #define GRASS_VERSION_RELEASE  @GRASS_VERSION_RELEASE@
```

Comment:
```suggestion
#define GRASS_VERSION_RELEASE  "@GRASS_VERSION_RELEASE@"
```

Reply:
note that this is can be also a number, eg. 7.8.**3**

Reply:
So it can be either a string (`dev`) or a number (e.g., `3`). Still, quoting this macro is safer and use `%s` not `%d` when printing it.